### PR TITLE
add mss role

### PR DESCRIPTION
**Pull request by wei-lee**
https://issues.jboss.org/browse/AEROGEAR-9808

### CODE REVIEW

**Comment by laurafitzgerald** [comment]
👀

File: roles/mss/templates/operator.yml.j2 [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  template:
+    metadata:
+      labels:
+        name: mobile-security-service-operator
+    spec:
+      serviceAccountName: mobile-security-service-operator
+      containers:
+        - name: mobile-security-service-operator
+          # Replace this with the built image name
+          image: {{ mobile_security_service_operator_image }}
```
Comment (laurafitzgerald) [inline roles/mss/templates/operator.yml.j2:19]: As we've just got the image tag here is it possible to import this file as we discussed yesterday? Based on the one operator manages one operand, I think we should.

Comment (wei-lee) [reply roles/mss/templates/operator.yml.j2:19]: yes, it should be possible. The only blocker for now is the same file in the repo always references the master tag. see https://github.com/aerogear/mobile-security-service-operator/blob/0.4.1/deploy/operator.yaml#L19.

Once that is fixed, we can import the file instead.

**Comment by wei-lee** [comment]
@laurafitzgerald are you happy for me to merge this PR for now, based on my response?

**Comment by laurafitzgerald** [comment]
@wei-lee sorry for the delay in response. I've tried to run this one a couple of times. Each time minishift tries to bring up everything together. I think because we don't have a wait in the ups and sso roles. Once it gets to mss there's a number of deployments together which don't get to a complete state and minishift dies then. Although I think the changes here would work fine.

EDIT: I've confirmed this by manually triggering the UPS and SSO deployments and they've both come up fine so I imagine it's a timing thing. Happy for this one to be merged as MSS came up fine after minishift came back but think we'll need to have waits added to UPS and SSO.

EDIT 2: Getting `0/1 nodes are available: 1 Insufficient memory` on SSO pod coming up. Had delete another project but it did come up after that.

**Comment by wei-lee** [comment]
@laurafitzgerald ok, I get what you mean. Yes, I will add some logic to wait for one service to complete before going to the next one. But I will add that login in separate PRs.

**Pull request merged**